src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -29,13 +32,15 @@ public PythonCodeMigrationEventArgs(string oldCode, string newCode)
 
     public abstract class PythonNodeBase : VariableInputNode
     {
-        private PythonEngineVersion engine = PythonEngineVersion.IronPython2;
+        private PythonEngineVersion? engine = null;
 
         [JsonConverter(typeof(StringEnumConverter))]
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Populate)]
+        [DefaultValue("IronPython2")]
         /// <summary>
         /// Return the user selected python engine enum.
         /// </summary>
-        public PythonEngineVersion Engine
+        public PythonEngineVersion? Engine
```

Comment:
I wonder if this qualifies as an API break...

Reply:
It does.

Reply:
that seems like a problem ;) Can we add a new enum like none?

Reply:
I can do this. I just felt we weren't tied to this API yet since it was recently added, but I guess people may be using it already.